simtbx/nanoBragg/nanoBragg.cpp

```diff
@@ -96,6 +96,12 @@ nanoBragg::nanoBragg(
     Fclose = Xclose = -dot_product(pix0_vector,fdet_vector);
     Sclose = Yclose = -dot_product(pix0_vector,sdet_vector);
     close_distance = distance =  dot_product(pix0_vector,odet_vector);
```

Comment:
I think `odet_vector` is equivalent to `detector[panel_id].get_normal()`. Also, I think `close_distance` is equivalent to `detector[panel_id].get_directed_distance()`. This does the dot product over on the dxtbx side. I prefer using the dxtbx library for calculations like this, because it then keeps application code shorter.

Reply:
I ultimately think it would be a good idea to write a test which verifies everything internal in nanoBragg lines up with dxtbx, if indeed one uses the dxtbx nanoBragg constructor.